src/clusterfuzz/_internal/metrics/logs.py

```diff
@@ -151,6 +175,8 @@ def get_logging_config_dict(name):
           get_handler_config('bot/logs/android_heartbeat.log', 1),
       'run_cron':
           get_handler_config('bot/logs/run_cron.log', 1),
+      'root':
```

Comment:
Why are we adding this?

Reply:
Unfortunately I can't recall how I hit this up, but it was when running some scripts, anyhow, I removed this, I'll do a dedicated PR if I get this issue again.